docs/how-to/release_process/canary.rst

```diff
@@ -0,0 +1,107 @@
+Canary Testing for Checkbox Edge Version: In-depth Process
+==========================================================
+
+Introduction
+------------
+
+The following sections provide a detailed walkthrough of the Canary Testing process for the Checkbox Edge version, starting from snap building to the outcomes of the testing.
+
+.. toctree::
+   :maxdepth: 1
+
+   canary_pipeline
+   validation_job_example
+   validation_pipeline_execution
+   
+
+Building and Publishing the Checkbox Snap
+-----------------------------------------
+
+Trigger Conditions
+^^^^^^^^^^^^^^^^^^
+The GitHub action responsible for building the snap runs every day. However, it only triggers when the following condition is met:
+
+At least one (PR) has been merged since the last edge build.
+
+Build URL
+^^^^^^^^^
+
+To monitor the build process, or to review the configuration and logs, visit the following URL:
+
+https://github.com/canonical/checkbox/actions/workflows/checkbox-snap-daily-builds.yml
+
+Post-Build Actions
+^^^^^^^^^^^^^^^^^^
+
+Once the build is successful:
+
+The snap packages are automatically pushed to the Snap Store.
+These snaps are released to the edge channel, allowing testers and early adopters to access the latest version.
+
+
+Jenkins Monitoring and Validation
+---------------------------------
+
+Snap Monitoring
+^^^^^^^^^^^^^^^
+
+Once the snap is published to the edge channel in the Snap Store, our Jenkins job titled `checkbox-edge-validation-detect-new-build` gets into action.
```

Comment:
Not sure if you want to italize the job name or to use the \<code> HTML tag as you do later in the document (in which case you should use double ticks \``, because .rst)

Reply:
Fixed by @tang-mm in the https://github.com/canonical/checkbox/pull/887.